webpack.renderer.config.js

```diff
@@ -0,0 +1,56 @@
+const rules = require('./webpack.rules')
+const plugins = require('./webpack.plugins')
+const path = require('path')
+const webpack = require('webpack')
+
+rules.push({
+  test: /\.less$/,
+  use: [{
+    loader: 'style-loader' // Create style nodes from JS strings
+  }, {
+    loader: 'css-loader' // Translate CSS into JS string
+  }, {
+    loader: 'less-loader' // Compile Less to CSS
+  }],
+  exclude: /theme-main\.less$/
+})
+rules.push({
+  test: /theme-main\.less$/,
+  use: [{
+    loader: 'style-loader', // Create style nodes from JS strings
+    options: { injectType: 'lazyStyleTag' } // Lazy-load themes so that we can switch between them
+  }, {
+    loader: 'css-loader' // Translate CSS into JS string
+  }, {
+    loader: 'less-loader' // Compile Less to CSS
+  }]
+})
+rules.push({
+  test: /\.handlebars$/,
+  use: [{
+    loader: 'handlebars-loader',
+    options: {
+      // Automatically load referenced images
+      inlineRequires: '/img/',
+      // Use custom Handlebars runtime with extra helpers registered
+      runtime: path.join(__dirname, 'source/common/zettlr-template.js')
```

Comment:
Short reminder that I said somewhere above that I think we should rename this thing to better reflect what it does now! :)